README.md

```diff
@@ -23,14 +23,13 @@ Install the addon.
 Import the mixins in a component and use the events or the handlers.
 
 ```javascript
-import Ember from 'ember';
+import Component from '@ember/component';
```

Comment:
This is supported by ember-cli-babel 6.6.0+. We have that as a dependency of the addon, so it works.